### PR TITLE
Update some links to .dev domains

### PR DESCRIPTION
**Pull request by parlough**
Updates some old `dartlang.org` urls to their new `.dev` counterparts.

### CODE REVIEW

**Comment by parlough** [comment]
@googlebot I signed it!

File: web/mobile.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <meta http-equiv="refresh" content="0;URL='https://dartpad.dartlang.org/index.html'" />    
   </head>    
   <body> 
-    <p>This page has moved to a <a href="https://dartpad.dartlang.org/index.html">
-      https://dartpad.dartlang.org/</a>.</p> 
+    <p>This page has moved to a <a href="https://dartpad.dev/index.html">
```
Comment (domesticmouse) [inline web/mobile.html:13]: I'd probably make this link to https://dartpad.dev/ instead. =)

**Comment by domesticmouse** [comment]
Hey @mit-mit can you check the dart.dev links in here are where you'd want them? I believe all the dartpad.dev links are sane.

**Review by domesticmouse — APPROVED** [review]

File: web/mobile.html (hunk 1 above)
Comment (parlough) [reply web/mobile.html]: Should be good now :)

**Review by mit-mit — APPROVED** [review]

**Review by domesticmouse — APPROVED** [review]
Thank you!

**Pull request merged**